trade-finance-manager-ui/server/controllers/case/amendments/amendmentRequestApproval.controller.js

```diff
@@ -31,6 +31,7 @@ const postAmendmentRequestApproval = async (req, res) => {
   const { facilityId, amendmentId } = req.params;
   const { requireUkefApproval } = req.body;
   const approval = requireUkefApproval === 'Yes';
```

Comment:
I think this value is `Boolean`?

Reply:
this value comes from the form. It's string at that time